{"status": "broken", "name": "Returning a list comprehension", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "skip"}, {"name": "feature", "value": "ListComprehension"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython2"}], "stop": 1578486036122, "historyId": "dac23cb1a799364b97840f4e891e7f15", "start": 1578486035920, "steps": [{"status": "passed", "start": 1578486035922, "stop": 1578486036119, "name": "Given an empty graph"}, {"status": "passed", "start": 1578486036119, "stop": 1578486036120, "name": "And having executed", "attachments": [{"source": "5cbf8a09-75ee-4899-8972-777961e24520-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "passed", "start": 1578486036121, "stop": 1578486036121, "name": "When executing query", "attachments": [{"source": "2ce64208-86ab-4234-a8df-220a95239f26-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "broken", "attachments": [{"source": "52c923f8-7c9c-412c-9b6f-0dddb82d3a23-attachment.csv", "type": "text/csv", "name": ".table"}], "stop": 1578486036122, "start": 1578486036121, "statusDetails": {"message": "ResponseError: RedisGraph does not currently support list comprehension\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "name": "Then the result should be"}, {"status": "skipped", "start": 1578486036122, "stop": 1578486036122, "name": "And no side effects"}], "statusDetails": {"message": "ResponseError: RedisGraph does not currently support list comprehension\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "uuid": "0f9f1cc7-77b6-4287-9f61-5bcc293f8dc5"}